contracts/lbp/LBPWrapper.sol

```diff
@@ -18,11 +18,13 @@ import "../utils/interface/IVault.sol";
 import "../utils/interface/ILBP.sol";
 
 contract LBPWrapper {
+    uint256 public constant swapFeePercentage = 1e12; // 0.0001% is minimum amount.
```

Comment:
Hmm, why we've changed variable to const? We need swapFeePercentage to be unchangeable?

Reply:
that still needs to be decided. As it will mostly used for fundraise, it was decided to have swap fee = 0. But LBP can have 0.0001% as the minimal swap fee. not less than that.

Reply:
First I thought that if we cannot set the fee to zero, we might want to add a functionality for the wrapper to claim the fees, but 0.0001% is really not worth the effort, I guess. If a project were to raise 1 million $ via the pool, the accrued fee would be just 1$. So probably we can live with some unclaimable dust.

Reply:
Ok, I see, then 0.00001 it is